.github/workflows/sbt-dependency-graph.yaml

```diff
@@ -1,19 +1,28 @@
-name: Update Dependency Graph for SBT
+name: Update Dependency Graph for sbt
 on:
   push:
     branches:
       - main
-  workflow_dispatch: 
+  workflow_dispatch:
 jobs:
   dependency-graph:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout branch
         id: checkout
-        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+        uses: actions/checkout@eef61447b9ff4aafe5dcd4e0bbf5d482be7e7871 # v4.2.1
+      - name: Install Java
+        id: java
+        uses: actions/setup-java@b36c23c0d998641eff861008f374ee103c25ac73 # v4.2.0
+        with:
+          distribution: corretto
+          java-version: 17
+      - name: Install sbt
+        id: sbt
+        uses: sbt/setup-sbt@8a071aa780c993c7a204c785d04d3e8eb64ef272 # v1.1.0
```

Comment:
Hi - could we use [`guardian/setup-scala`](https://github.com/guardian/setup-scala) instead? This project already has `.tool-versions` file, and from @adamnfish's email on 4th December, DevX do plan to use `guardian/setup-scala`:

> To summarise, we are replacing the standard actions/setup-java step with our own Guardian-managed guardian/setup-scala step. This new step installs Java and Scala (sbt) and simplifies the configuration of our runners by aligning with other parts of our tooling. In particular, the Java version is no longer provided directly in the workflow file. Instead, this comes from a .tool-versions file at the root of the repository.

I'm happy to make the changes (also following on from https://github.com/guardian/play-googleauth/pull/275 & https://github.com/guardian/simple-configuration/pull/103), just don't want to clash if there's a reason to go another way!?

Reply:
Go for it! I only didn't use it because the example I was copying from (raised by the dependency integrator bot) didn't use setup-scala – I don't have any strong feelings about it.

Reply:
(Looking at dates, the integrator bot raised the PR I copied from on the 2nd of December, so presumably it just predates the decision to use setup-scala)